### DISCUSSION TITLE
Any way to ignore certain service roads for roundabout exit number?

### DISCUSSION THREAD

**Discussion by ntzm** [valhalla/valhalla#4262] [Q&A] [answered]
In this situation

https://www.openstreetmap.org/directions?engine=fossgis_valhalla_car&route=51.60125%2C-0.63614%3B51.60159%2C-0.63872

Valhalla says to take the second exit, whereas OSRM and other routing such as Google would tell you to take the first exit. OSRM seems to ignore all service roads, which I don't like the behaviour of either. Is there a way of only ignoring certain service roads for roundabout exit numbers? In this example, ignoring parking isles would be helpful.

Thanks

**Comment by nilsnolde** [comment, marked as answer]
You're sure about Google? Seems it's also calling out as 2nd exit: https://www.google.com/maps/dir/51.6015403,-0.6366393/51.6016566,-0.6378563/@51.6017064,-0.638531,18z/am=t?entry=ttu

OSRM says "1st exit" for _both_ parking aisle and the proper road when set as destination. IMO that's much weirder. I think as long as there's connectivity and you can actually exit the roundabout into the parking aisle, it's worth counting as a real exit.

**Reply by ntzm** [reply]
Ah you're right. Thanks. I think most people will expect it to say 1st exit in this scenario however, but I think the road layout is mostly to blame.

**Reply by nilsnolde** [reply]
Yeah agree, that's very confusing to a driver who's never sure if those tiny roads are actually counting. Similar problem to those new turbo roundabouts in Europe.
